nzgoal/NZGOAL-Version-2.md

```diff
@@ -143,35 +139,29 @@ NZGOAL Review and Release Process Decision Trees	32
 
-experts and others can contribute to improved policy development and more efficient financial performance by government through being able to access, manipulate and provide feedback on such material. 
+    (b) experts and others can contribute to improved policy development and more efficient financial performance by government through being able to access, manipulate and provide feedback on such material. 
 
-NZGOAL provides a means of realising this potential. It:
+ 2. NZGOAL provides a means of realising this potential. It:
 
-sets out a series of open licensing and open access principles, for copyright works and non-copyright material respectively;
+    (a) sets out a series of open licensing and open access principles, for copyright works and non-copyright material respectively;
 
-advocates the use of:	
+    (b) advocates the use of:	
 
-Creative Commons licences for those State Services agencies’ copyright works which are appropriate for release and re-use; and
+        (i) Creative Commons and Open Source licences for those State Services agencies’ copyright works which are appropriate for release and re-use; and
 
-clear “no known rights” statements for non-copyright material released for re-use; and
+        (ii) clear “no known rights” statements for non-copyright material released for re-use; and
 
-sets out a review and release process to guide agencies through the review of works and other material they consider ought to be released for re-use.  
+    (c) sets out a review and release process to guide agencies through the review of works and other material they consider ought to be released for re-use.  
 
-Version 1 of NZGOAL was released in 2010. This is version of 2 of NZGOAL. Primarily it refines and simplifies NZGOAL and updates the code samples, for online releases of Creative Commons licensed works, to HTML 5. It also recommends the Creative Commons 4.0 International licences in preference to the Creative Commons 3.0 New Zealand licences but without requiring any retrospective action in relation to works already licensed under the 3.0 licences. 
+ 3.  Version 1 of NZGOAL was released in 2010. Version of 2 of NZGOAL was released in 2014. This is version 3 of NZGOAL. Primarily it continues to refine and simplify NZGOAL and expands it's scope to provide guidance for the release and reuse of Open Source software copyright material. The licensing considerations of Open Source Software and wider social, cultural and economic benefits of release and reuse of such software is virually identical to those of open data and content. Primary differences include the wider range of licenses used when releasing Open Source Software for reuse by others and the ongoing collaborative process that releasing Open SOurce Software can foster. 
```

Comment:
I have rewritten this section as though a fictitious version 3 of NZGOAL were to be released that simply expanded scope to OSS.